field.h

```diff
@@ -0,0 +1,36 @@
+#ifndef FIELD_H
+#define FIELD_H
+#include "cell.h"
+#include <vector>
+#include <list>
+
+class Field{
+public:
+    //Field();
+    Field(int fieldSize = 5);
+    int getSize(){return size;}
+
+    std::vector<std::vector<Cell>>& getField(){return field;}
+    bool getMovableStatus(){return isMovable;}
+
+    void updateMovability(); //Update field movability
```

Comment:
Update movability should be private I guess

Reply:
done